Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
I don't get the part about manifest, sorry. In a which way it could be used by depot_tools to fetch v8?